<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>医院管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 主窗体背景色 */
QMainWindow {
    background-color: #F5E6D3;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">/* --- 明亮暖色调医院管理系统主题 --- */

/* Main Window &amp; Central Widget */
QMainWindow, QWidget#centralwidget {
    background-color: #F5E6D3;
}

/* --- Frame --- */
QWidget#frame
{
	background-color: #F5E6D3;
}
/* --- Sidebar --- */
QWidget#sidebar {
    background-color: #E8D5C4;
	border-radius: 10px;
	border: 1px solid #D2B48C;
}
QWidget#sidebar QPushButton {
    background-color: transparent;
    border: none;
    color: #8B4513;
    padding: 10px;
    padding-left: 15px;
    border-radius: 5px;
    text-align: left;
    font-weight: bold;
}
QWidget#sidebar QPushButton:hover {
    background-color: #DEB887;
}
QWidget#sidebar QPushButton:pressed,
QWidget#sidebar QPushButton:checked {
    background-color: #FF8C42;
    color: #FFFFFF;
}

/* --- Topbar --- */
QFrame#topbar {
    background-color: #E8D5C4;
    border-bottom: 1px solid #D2B48C;
}
}
QFrame#topbar QPushButton {
    background-color: transparent;
    border: none;
    border-radius: 4px;
    padding: 5px;
    text-align: center;
    color: #8B4513;
}
QFrame#topbar QPushButton:hover {
    background-color: #DEB887;
}
QFrame#topbar QPushButton#close:hover {
    background-color: #CD853F;
}

/* --- Main Content Area --- */
QFrame#content, QStackedWidget &gt; QWidget {
    background-color: #F5E6D3;
    border: none;
}

/* --- Stacked Widget Background --- */
QStackedWidget#contentStackWidget {
    background-color: #F5E6D3;
}

/* --- Home Page Background --- */
QWidget#homePage {
    background-color: #F5E6D3;
}

/* --- Layout Backgrounds --- */
QGridLayout#statsGrid {
    background-color: #F5E6D3;
}

QVBoxLayout#verticalLayout_2 {
    background-color: #F5E6D3;
}

/* General Page Titles */
QLabel#pageTitle {
    color: #8B4513;
    font-size: 18pt;
    font-weight: bold;
}

/* --- Dashboard Specific Styles --- */
#homePage .QFrame {
    background-color: #FFFFFF;
    border-radius: 12px;
    border: 1px solid #D2B48C;
    box-shadow: 0 2px 4px rgba(139, 69, 19, 0.1);
}
#homePage QLabel {
    color: #8B4513;
    font-size: 11pt;
}
#homePage #mainChartTitle, #homePage #barChartTitle {
    font-size: 14pt;
    font-weight: bold;
    padding: 5px;
}
#homePage QProgressBar {
    border: 1px solid #D2B48C;
    border-radius: 5px;
    text-align: center;
    color: #8B4513;
    background-color: #F5E6D3;
}
#homePage QProgressBar::chunk {
    background-color: #FF8C42;
    border-radius: 4px;
}
#homePage #op1Button, #homePage #op2Button {
    background-color: #FF8C42;
    color: #FFFFFF;
    border-radius: 5px;
    padding: 5px 10px;
    font-weight: bold;
}

/* --- Patients &amp; Appointments Page Styles --- */
#patientsPage QPushButton, #appointmentsPage QPushButton {
    background-color: #FF8C42;
    color: #FFFFFF;
    border-radius: 5px;
    padding: 8px 12px;
    font-weight: bold;
}
#patientsPage QPushButton:hover, #appointmentsPage QPushButton:hover {
    background-color: #FF9F5A;
}

/* --- Table View --- */
QTableView {
    background-color: #FFFFFF;
    border: 1px solid #D2B48C;
    border-radius: 8px;
    gridline-color: #D2B48C;
    color: #8B4513;
}
QTableView::item {
    padding: 5px;
    border-bottom: 1px solid #D2B48C;
}
QTableView::item:selected {
    background-color: #FF8C42;
    color: #FFFFFF;
}
QHeaderView::section {
    background-color: #E8D5C4;
    color: #8B4513;
    padding: 5px;
    border: 1px solid #D2B48C;
    font-weight: bold;
}

/* --- Line Edit (Search Bar) --- */
QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #D2B48C;
    border-radius: 5px;
    padding: 8px 10px;
    color: #8B4513;
}
QLineEdit:focus {
    border-color: #FF8C42;
}

/* --- Calendar Widget --- */
QCalendarWidget QToolButton {
    color: #8B4513;
    background-color: transparent;
    border: none;
    font-size: 14px;
    font-weight: bold;
}
QCalendarWidget QToolButton:hover {
    background-color: #DEB887;
}
QCalendarWidget QMenu {
    background-color: #FFFFFF;
    border: 1px solid #D2B48C;
}
QCalendarWidget QSpinBox {
    color: #8B4513;
    background-color: #FFFFFF;
    border: 1px solid #D2B48C;
}
QCalendarWidget QWidget#qt_calendar_navigationbar {
    background-color: #E8D5C4;
    border: 1px solid #D2B48C;
    border-bottom: none;
    border-radius: 8px 8px 0 0;
}
QCalendarWidget QAbstractItemView {
    background-color: #FFFFFF;
    color: #8B4513;
    selection-background-color: #FF8C42;
    selection-color: #FFFFFF;
    border: 1px solid #D2B48C;
    border-top: none;
    border-radius: 0 0 8px 8px;
}
QCalendarWidget QAbstractItemView:disabled {
    color: #CD853F;
}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="sidebarLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QPushButton" name="toggleButton">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string>医院系统</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/health.png</normaloff>:/img/health.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="homeButton">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string>仪表盘</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/Dashboard.png</normaloff>:/img/Dashboard.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="patientsButton">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string>患者管理</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/Patients.png</normaloff>:/img/Patients.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="medicalRecordsButton">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string>病历管理</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/Report.png</normaloff>:/img/Report.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="appointmentsButton">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string>预约管理</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/Appointments.png</normaloff>:/img/Appointments.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="settingsButton">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string>系统设置</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/img/Settings.png</normaloff>:/img/Settings.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="topbar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/img/minus.svg</normaloff>:/img/minus.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/img/maximize.svg</normaloff>:/img/maximize.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="close">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/img/x.svg</normaloff>:/img/x.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="content">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QStackedWidget" name="contentStackWidget">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="homePage">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QGridLayout" name="statsGrid">
                <item row="0" column="0" colspan="2">
                 <widget class="QFrame" name="mainChartFrame">
                  <layout class="QVBoxLayout" name="mainChartLayout">
                   <item>
                    <layout class="QHBoxLayout">
                     <item>
                      <widget class="QLabel" name="mainChartTitle">
                       <property name="text">
                        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#FF8C42;&quot;&gt;心电监护&lt;/span&gt; 实时监测&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer>
                       <property name="orientation">
                        <enum>Qt::Orientation::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>0</width>
                         <height>0</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QPushButton" name="op1Button">
                       <property name="text">
                        <string>开始监护</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="op2Button">
                       <property name="text">
                        <string>暂停监护</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QGraphicsView" name="mainChartPlaceholder">
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="2" colspan="2">
                 <widget class="QFrame" name="barChartFrame">
                  <layout class="QVBoxLayout" name="barChartLayout">
                   <item>
                    <widget class="QLabel" name="barChartTitle">
                     <property name="text">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#FF8C42;&quot;&gt;心率&lt;/span&gt; 72 BPM&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QGraphicsView" name="barChartPlaceholder">
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QFrame" name="donutChartFrame">
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#FF8C42;&quot;&gt;血压&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt; 120/80 mmHg&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QGraphicsView" name="donutChartPlaceholder">
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QFrame" name="verticalBarFrame">
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#FF8C42;&quot;&gt;血氧&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt; 98% SpO2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QGraphicsView" name="verticalBarPlaceholder">
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>

               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="patientsPage">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="pageTitle">
                <property name="text">
                 <string>患者信息管理</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="addPatientButton">
                  <property name="text">
                   <string>添加新患者</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Orientation::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="searchLineEdit">
                  <property name="placeholderText">
                   <string>按姓名或编号搜索患者...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTableView" name="patientsTableView"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="appointmentsPage">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="appointmentsPageTitle">
                <property name="text">
                 <string>预约时间安排</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="spacing">
                 <number>12</number>
                </property>
                <item>
                 <widget class="QCalendarWidget" name="calendarWidget">
                  <property name="locale">
                   <locale language="English" country="FalklandIslands"/>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="styleSheet">
                     <string notr="true">font-size: 14pt; font-weight: bold;</string>
                    </property>
                    <property name="text">
                     <string>预约列表</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QTableView" name="appointmentsTableView"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="addAppointmentButton">
                    <property name="text">
                     <string>新建预约</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Orientation::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Policy::Expanding</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
